data/links.yaml

```
@@ -12,34 +12,38 @@ novice:
   setup: ""
   install: "https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html#installing-and-upgrading-ansible"
   start: "https://docs.ansible.com/ansible/latest/getting_started/index.html"
-user_actions:
-  automate: "https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_intro.html#playbook-syntax"
-  evaluate_plugins: "https://docs.ansible.com/ansible/latest/module_plugin_guide/modules_intro.html"
-  build_inventory: "https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html"
 maintainer_actions:
   overview: "https://docs.ansible.com/ansible/latest/community/contributor_path.html"
   learn: "https://docs.ansible.com/ansible/latest/community/maintainers_guidelines.html"
   join: "https://docs.ansible.com/ansible/latest/community/maintainers_guidelines.html#weekly-community-matrix-irc-meetings"
-project_doc_home:
-  als: "https://als.readthedocs.io/en/latest/"
-  awx: "https://github.com/ansible/awx"
-  awx_operator: "https://github.com/ansible/awx-operator"
-  builder: "https://ansible-builder.readthedocs.io/en/latest/"
-  compat: "https://ansible-compat.readthedocs.io/en/latest/"
-  eda: "https://github.com/ansible/eda-server"
-  galaxy: "https://galaxy.ansible.com/docs/"
-  galaxy_ng: "https://galaxyng.netlify.app/"
-  lint: "https://ansible-lint.readthedocs.io/en/latest/"
-  molecule: "https://molecule.readthedocs.io/en/latest/"
-  navigator: "https://ansible-navigator.readthedocs.io/en/latest/"
-  pinakes: "https://github.com/ansible/pinakes"
-  pylibssh: "https://ansible-pylibssh.readthedocs.io/en/latest/"
-  receptor: "https://receptor.readthedocs.io/en/latest/index.html"
-  rulebook: "https://ansible-rulebook.readthedocs.io/en/latest/"
-  runner: "https://ansible-runner.readthedocs.io/en/stable/"
-  sdk: "https://ansible-sdk.readthedocs.io/en/latest/"
-  sign: "https://ansible.github.io/ansible-sign/"
-  vscode: "https://marketplace.visualstudio.com/items?itemName=redhat.ansible"
+# Milestone actions for the Ansible user persona.
+user:
+  automate:
+    start: "https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_intro.html#playbook-syntax"
+    evaluate_plugins: "https://docs.ansible.com/ansible/latest/module_plugin_guide/modules_intro.html"
+  build_inventory:
+    multiple_hosts: "https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html"
+    dynamic: "https://docs.ansible.com/ansible/latest/inventory_guide/intro_dynamic_inventory.html"
+  organize:
+    roles: "https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_reuse_roles.html"
+  find:
+    explore_galaxy: "https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#finding-collections-on-galaxy"
+    install_roles: "https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-roles-from-galaxy"
+    install_collections: "https://docs.ansible.com/ansible/latest/galaxy/user_guide.html#installing-collections"
+    customize_content: ""
```

```suggestion
    customize_content: ""
```
is it missed intentionally?

---

@Andersson007 Yes I couldn't find that and will add it to the issue tracking content gaps. https://github.com/ansible/jinja-docsite/issues/32 

Have you got a suggestion for the link?

---

I'll just add it to that issue and we can go from there.